lua/catppuccin/groups/integrations/fzf.lua

```diff
@@ -2,9 +2,9 @@ local M = {}
 
 function M.get()
 	return {
-		FzfLuaNormal = { link = "NormalFloat" },
+		-- FzfLuaNormal = { link = "NormalFloat" },
```

Comment:
Is there a reason this should stay commented out? 🤔

Reply:
I saw that telescope integration [did the same](https://github.com/catppuccin/nvim/blob/5d9582dbbe37e465f7f93ca6e3dd60cd20eea44f/lua/catppuccin/groups/integrations/telescope.lua#L48). lmk if it needs a little explanation or if I just remove it

Reply:
Ah didn't realise Telescope had the same. I think a little comment/explanation like the one on Telescope would be good!